### DISCUSSION TITLE
AC Voltage Anzeige nach Update höher?

### DISCUSSION THREAD

**Discussion by ce3rd** [tbnobody/OpenDTU#301] [Q&A] [answered]
Hoi,

kanns sein, dass Voltage (AC Seite) nun irgendwie anders "berechnet" wird? Ich habe gestern Update auf 9ee75ed gemacht und seither ist Voltage immer > 240V - ich hab das leider vorher nicht gegrapht - aber bilde mir ein, das war vorher nie so - hab auch paar alte Werte gefunden - die waren  max. 234V

"Voltage":{"v":242.1000061,"u":"V","d":1}

Ich hätte mir zwar gedacht, das spuckt direkt der Inverter aus, aber was weiss ich ;-)

Danke! Und auch Danke für das tolle Projekt, bin so froh, dass ich nicht diese hoymiles Cloud verwenden musss...

**Comment by tbnobody** [comment, marked as answer]
Die Spannung wird nicht berechnet. Die kommt 1zu1 so aus dem Inverter. Hier wurde auch nichts umgestellt.

**Comment by ce3rd** [comment]
ok, danke für die schnelle antwort, is dann wohl eh normal und is mir bis jetzt noch nie aufgefallen... 
ist ja auch innerhalb der grenzwerte im gridprofile 

und vielleicht noch ne blöde frage: gibts eigentlich die möglichkeit via opendtu das gridprofile auszulesen/einzustellen? bzw is sowas geplant?

**Reply by tbnobody** [reply]
Aktuell gibt es keine Möglichkeit das GridFile zu lesen / zu schreiben. Ein auslesen und parsen der Werte würde mich ggf. noch reizen aber setzen halte ich für sehr gefährlich. v.a. wenn man 1. nicht weiß was man tut und 2. wenn beim experimentieren irgendetwas schief geht. Damit kann man sich im einfachsten Fall den eigenen Inverter zerstören, im schlimmeren Fall unzulässige Ströme ins Netz einspeisen.

**Reply by Sinned42** [reply]
Ich glaube das Netz ist so stabil, dass man keine unzulässigen Ströme generieren kann, die das Netz juckt :) Also wird der Inverter als schwächstes Glied der Kette dicke Backen machen, danach evtl. die Sicherung für die Steckdose, ähh, ich meine natürlich die Wielanddose, an dem das Ding hängt.

**Reply by ce3rd** [reply]
auslesen wär auf jedenfall ein bringer, denk ich, weil zb hier in österreich muss man angeblich für den konformen betrieb so TOR-settings einspielen und falls mal der liebe netzbetreiber fragt, kann man das dann auch herzeigen. ich weiss nicht wie das in DE ist und ich hab auch keine ahnung wie sich die settings von einem "nackerten" inverter unterscheiden, hab anfänglich, bevor ich opendtu kannte, auf jedenfall über die originale DTU und die cloud das profile reingeknallt -> AT_TOR_Erzeuger_default, weil das eben angeblich hier so sein muss... 

kann aber auch sein, dass  ich  kompletten müll schreib :-) beruht rein auf hörensagen

**Reply by ce3rd** [reply]
extra senf hierzu: das hier könnte darauf rauslaufen, dass auch die möglichkeit zum setzen der werte ev. sinn machen würde/wird:

https://www.photovoltaikforum.com/thread/181890-anlage-mit-mehreren-hoymiles-hm-1500-mikrowechselrichtern-anmelden/?postID=2855729#post2855729

**Reply by stefan123t** [reply]
Hatte jetzt auch schon zwei Wortmeldungen bzgl. WR aus AT die auf 70% begrenzt sind. Das würde ja zumindest für das Auslesen des GridProfile sprechen um das zu verfizieren. Evtl könnte man auch die unterschiedlichen GridProfiles zur Verfügung stellen ?
